pl.json

```diff
@@ -1769,7 +1769,7 @@
         "Settings.CursorSettings.GrabMultiplier.Description": "Jest stosowany jako mnożnik rozmiaru kursora podczas chwytania obiektów i przedmiotów.",
 
         "Settings.CursorSettings.InteractionMultiplier": "Mnożnik interakcji",
-        "Settings.CursorSettings.InteractionMultiplier.Description": "Jest stosowany jako mnożnik rozmiaru kursora podczas while interacting with UI elements such as buttons.",
+        "Settings.CursorSettings.InteractionMultiplier.Description": "Jest stosowany jako mnożnik rozmiaru kursora podczas podczas interakcji z elementami interfejsu użytkownika, takimi jak przyciski.",
```

Comment:
Oh dear. The saga continues! Did not have enough coffee when I allegedly fixed this...